man/coredumpctl.xml

```diff
@@ -227,11 +235,21 @@
         <option>--output=</option>. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><command>debug</command></term>
+
+        <listitem><para>Invoke the a debugger on the last core dump
+        matching specified characteristics. The debugger used (either gdb or lldb)
+        will first be read from <option>--debugger</option>, then from
```

Comment:
What's the reason to restrict this to only `gdb` or `lldb`? It seems that this can simply be the name of a generic executable.